llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/base.py

```diff
@@ -281,6 +286,7 @@ def __init__(
             pydantic_program_mode=pydantic_program_mode,
             output_parser=output_parser,
             strict=strict,
+            supports_content_blocks=model in ALL_AVAILABLE_MODELS,
```

Comment:
As more llms add support, we can change how this works. For now, only supported for openai models